LICENSE.darrenscerri

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2017 Darren Scerri
```

Comment:
2019, not 2017 😄

Reply:
actually, do we need this one?

Reply:
This is the license from the other project. We need to include it here untouched. I stuck the owner's name on the end as the extension so that we could have our own LICENSE file without conflicts.